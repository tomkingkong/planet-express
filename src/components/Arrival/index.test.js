import React from 'react';
import { shallow } from 'enzyme';

import { Arrival } from '.';

describe('Arrival component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Arrival />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should send the user to the home page on click', () => {
    const mockFn = jest.fn();
    const history = {replace:mockFn};
    const wrapper = shallow(<Arrival history={history} />);
    wrapper.find('.retire_btn').simulate('click');
    expect(mockFn).toHaveBeenCalled();
  });
});