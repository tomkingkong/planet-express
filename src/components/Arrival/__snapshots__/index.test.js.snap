// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrival component should match snapshot 1`] = `
<article
  className="Arrival"
>
  <h3
    className="arrival__title"
  >
    You have arrived!
  </h3>
  <div
    className="space"
  >
    <Component />
  </div>
  <button
    className="retire_btn"
    onClick={[Function]}
  >
    Click to return Home
  </button>
</article>
`;
